sys/mips/include/atomic.h

```diff
@@ -90,8 +90,11 @@ static __inline int atomic_fcmpset_8(__volatile uint8_t *, uint8_t *, uint8_t);
 static __inline int atomic_cmpset_16(__volatile uint16_t *, uint16_t, uint16_t);
 static __inline int atomic_fcmpset_16(__volatile uint16_t *, uint16_t *, uint16_t);
 
-static __inline void
 /* Work around https://github.com/CTSRD-CHERI/qemu/issues/4 */
+#define QEMU_TLB_WORKAROUND8(register) \
```

Comment:
Do we still need these? Don't really want to be adding more workarounds, and I think the existing ones should be long overdue for removal.

Reply:
Yeah these should go. No longer required anymore because we fixed QEMU to return to the correct address on trap a long time ago.